apidoc/Titanium/Media/AudioPlayer.yml

```diff
@@ -426,7 +542,10 @@ properties:
     platforms: [iphone, ipad]
 
   - name: bufferSize
-    summary: Size of the buffer used for streaming, in bytes.
+    summary: Size of the buffer used for streaming, in milliseconds.
```

Comment:
I don't think we should change `bufferSize` from bytes to milliseconds. Especially since we already have a `duration` property which provides this.

Reply:
I may missing some info here: Can we convert milliseconds to bytes? Is "bytes" simply "BPM"?

Reply:
Does AVPlayer provide the bitrate? If so, then you can use that in combination with duration to calculate the byte buffer size.

Also, while some audio formats such as MP3 have a variable bitrate, an audio player typically decodes it to a fixed bitrate (say 16bit) in PCM form before playing it. This is the buffer size we're after since it indicates how much RAM the decoded audio file is consuming (assuming that we're not streaming).